logic/auth.js

```diff
@@ -51,33 +68,58 @@ async function changePassword(currentPassword, newPassword, jwt) {
     changePasswordStatus.percent = 1; // eslint-disable-line no-magic-numbers
 
     try {
-      // update user file
-      const user = await diskLogic.readUserFile();
-      const credentials = hashCredentials(SYSTEM_USER, newPassword);
+        // update user file
+        const user = await diskLogic.readUserFile();
+        const credentials = hashCredentials(SYSTEM_USER, newPassword);
 
-      // re-encrypt seed with new password
-      const decryptedSeed = await iocane.createSession().decrypt(user.seed, currentPassword);
-      const encryptedSeed = await iocane.createSession().encrypt(decryptedSeed, newPassword);
+        // re-encrypt seed with new password
+        const decryptedSeed = await iocane.createSession().decrypt(user.seed, currentPassword);
+        const encryptedSeed = await iocane.createSession().encrypt(decryptedSeed, newPassword);
 
-      // update user file
-      await diskLogic.writeUserFile({ ...user, password: credentials.password, seed: encryptedSeed });
+        // update user file
+        await diskLogic.writeUserFile({ ...user, password: credentials.password, seed: encryptedSeed });
 
-      // update system password
-      await setSystemPassword(newPassword);
+        // update system password
+        await setSystemPassword(newPassword);
```

Comment:
Maybe reverting these changes would help getting this merged. I recommend to not lint any of your PRs, Umbrel hasn't used a configured linter in any project for over a year now, even though that could've prevented some bugs.